src/SourceBuild/tarball/content/eng/bootstrap/README.template.md

```diff
@@ -120,6 +121,20 @@ The dependencies for building .NET from source can be found [here](https://githu
     dotnet --info
     ```
 
+### Building using Docker
+
+You can also build the repository using a Docker image which has the required prerequisites inside.
+The example below creates a Docker volume named `vmr` and clones and builds the VMR there.
+
+```sh
+docker run --rm -itv vmr:/vmr -w /vmr mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8
```

Comment:
```suggestion
docker run --rm -it v vmr:/vmr -w /vmr mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8
```

Reply:
Should this be then this?
```suggestion
docker run --rm -it -v vmr:/vmr -w /vmr mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8
```

Reply:
Its a preference.  I find it more flexible when providing instructions.